### ISSUE TITLE
No Action on clicking Quick Buy on the Product Page

### ISSUE THREAD

**Issue by planetoid2001** [varunsridharan/woocommerce-quick-buy#38]
Hi Varun,

Hope you are doing well!

I've encountered an issues since a few days wherein on clicking the Quick Buy button, nothing happens. Besides that after clicking on the Quick buy button, all the other buttons on the page stop functioning.

Prior to this the issue I was facing is that on clicking the Quick buy button, the product was getting added to the cart instead of redirecting to the Checkout Page

Could you please help me out with this

Thanks

**Comment by varunsridharan** [comment]
@planetoid2001 Can you please check if your site is generating any javascript errors ?

### How To Open Browser Console
The Browser Console contains important information on Errors. While most of the errors shown are JavaScript related some server related information can get displayed as well.
Chrome:

    Press either CTRL + SHIFT + J to open the “console” tab of the Developer Tools.

#### Alternative method:

    Press either CTRL + SHIFT + I or F12 to open the Developer Tools.
    Press ESC (or click on “Show console” in the bottom right corner) to slide the console up.

Note: In Chrome’s dev tools, there is a “console” tab. However, a smaller “slide-up” console can be opened while any of the other tabs is active.

#### Safari:

    Press CTRL + ALT + I to open the Web Inspector.
    See Chrome’s step 2. (Chrome and Safari have pretty much identical dev tools.)

Note: Step 1 only works if the “Show Develop menu in menu bar” check box in the Advanced tab of the Preferences menu is checked!

#### IE9:

    Press F12 to open the developer tools.
    Click the “console” tab.

#### Firefox:

    Press CTRL + SHIFT + K to open the Web console (COMMAND + SHIFT + K on Macs).

> or, if Firebug is installed (recommended):

    Press F12 to open Firebug.
    Click on the “console” tab.

#### Opera:

    Press CTRL + SHIFT + I to open Dragonfly.
    Click on the “console” tab.

**Comment by planetoid2001** [comment]
Hi @varunsridharan 

There doesn't seems to be any errors. I just checked the console while on the product Page with the plugin active. No Errors and No warnings.
![Capture](https://user-images.githubusercontent.com/69184294/89261694-1e27ba00-d64c-11ea-966e-005f5b1c4bab.PNG)


Thanks

**Comment by ducanhit066** [comment]
I have same problem

**Comment by varunsridharan** [comment]
@ducanhit066 @planetoid2001 i tried to replicate the same issue but working fine for me.

Would you mind sharing your website url where this issue is present ?

**Comment by planetoid2001** [comment]
Hi @varunsridharan 

Sorry for the Delay, below is a second site that I made for testing purposes and I am facing the same issue with the quick buy button ... 

Also, how exactly do I private reply here? :) 
Thanks!

**Comment by dvector** [comment]
@planetoid2001 Your ADD TO CART button also appears to be not working. Both buttons work when I remove the following script `wp-content/themes/oceanwp/assets/js/third/woo/woo-ajax-add-to-cart.min.js?ver=1.8.6` (seen on line 1029 view-source) So its the theme you are using OceanWP that is loading its own ajax-add-to-cart handler and is conflicting?

**Comment by planetoid2001** [comment]
Let me check and get back to you. So your saying that if I have an Ajax add
to cart enabled, then the quick buy button may not work? :( I like Ajax

Btw, the Add to cart works only if that's the first button you click after
the page loads ( without clicking the Buy Now button)

Any way this can be fixed in your next release?

Mel

**Comment by varunsridharan** [comment]
@dvector Thanks for your support :-)

@planetoid2001 can you please share the addtocart ajax plugin download link so i can check & write integration if it requires.

**Comment by planetoid2001** [comment]
@varunsridharan Thanks for looking into this!

There's no separate plugin ... its a feature available in the woocommerce customiser ... WooCommerce -> Single Product -> ENABLE AJAX ADD TO CART. I double checked on this and unchecking this option definitely makes your plugin work. So, the problem seems to be the OceanWP / WooCommerce colab I guess. There is an unrelated conversation on a similar problem with Ajax on the oceanwp support on wordpress.org, pasting the link here just in case it helps 
https://wordpress.org/support/topic/what-does-oceanwp-change-in-the-add-to-cart-process/

Looks like there might be more more who have faced this issue but have probably not reported it and may have just disabled Ajax for Add to Cart on Single product (if they figured it out that is) :) or worse may have removed your plugin which is a shame because your plugin is simply amazing.

Just wanted to mention that your support is awesome. Great Going Varun. 

Thanks!
Mel

<blockquote><img src="//s.w.org/favicon.ico?2" width="48" align="right"><div><strong><a href="https://wordpress.org/support/topic/what-does-oceanwp-change-in-the-add-to-cart-process/">What does OceanWP change in the add to cart process? | WordPress.org</a></strong></div><div>[This thread is closed.] Hi, i have a problem with oceanwp and a plugin called PayPal Plus for Woocommerce. The error occurs when the plugin calls a…</div></blockquote>

**Comment by octoniondesign** [comment]
Hi Varun,

Hope you are doing well!

woocommerce quick buy button not working in IE11.

Could you please help me out with this

Thanks
Mel

**Comment by javionniu** [comment]
Hi @varunsridharan 

I ran into the same problem, but when I disabled /wp-content/plugins/woocommerce/assets/js/frontend/add-to-cart-variation.js through the console, the button resumed its function.

I am using Wordpress 5.5
Woocommerce Quick Buy 2.8.1
![image](https://user-images.githubusercontent.com/14959392/92188675-d3d66a80-ee8f-11ea-9d1a-d89bbb11171c.png)

**Comment by varunsridharan** [comment]
@javionniu 
Yes it will work if you disable it. but the isuse dose not occur in all the sites.

It would be great if you can share the site's link so i can check what kind of issue is that. using which i can try to fix it in the next update.
